opensrp-chw/src/kk/assets/json.form/add_anc_registration.json

```diff
@@ -366,7 +362,7 @@
         "calculation": {
           "rules-engine": {
             "ex-rules": {
-              "rules-file": "add_anc_registration_calculation.yml"
+              "rules-file": "add_anc_registration_relevance.yml"
```

Comment:
Do you have the calculations in the relevance file?

Reply:
Yes, to improve the performance ya hiyo form, realized with those calculations if you have relevance and calculations on different files the form becomes really slow